packages/react-reconciler/src/ReactCurrentFiber.js

```diff
@@ -37,7 +37,9 @@ function describeFiber(fiber: Fiber): string {
     default:
       const owner = fiber._debugOwner;
       const source = fiber._debugSource;
-      const name = getComponentName(fiber.type);
+      const name = fiber.type
+        ? getComponentName(fiber.type)
+        : getComponentName(fiber.elementType);
```

Comment:
Hm... I think this change wouldn't do what your PR description says:

> The `displayName` support may be leveraged in a Babel plugin (see https://github.com/babel/babel/pull/10201) to infer the name from context and make this seamless for most users:

```diff
 const Foo = React.lazy(() => import('./FooImplementation'));
+Foo.displayName = 'Foo';
```

It *would* display "Lazy" rather than "Unknown", which would be an improvement, but to actually honor the `displayName` (if we wanted to) would also require a change to `getComponentName`)